### ISSUE TITLE
Fix Keyboard Accessibility for Top Tabs

### ISSUE THREAD

**Issue by russtuck** [gordon-cs/gordon-360-ui#1390]
The non-home tabs on 360 are not accessible via keyboard. Using <tab> to access them fails because focus jumps straight to the search box (and never comes back).  

The arrow keys should also work to move to these tabs.

**Comment by bennettforkner** [comment]
So the arrow keys work if you <kbd>TAB</kbd> to the nav menu, then use them. Are you seeing this same functionality @russtuck ?

**Comment by russtuck** [comment]
Yes, that does work.  And if I turn on Voiceover (the Mac screen reader),
it reads it clearly that way.  So it's discoverable for people who use a
screen reader because of vision issues, but less so for people who use
keyboard navigation for other reasons like limited dexterity (though they
might figure it out faster), or pure convenience (like me).

So I agree it's not a bug.  It might even be the best choice.  I think I'll
copy Simon Miner, Gordon alumnus and ChristianBook web architect and
accessibility guru, to see if he has an opinion.

   Russ

**Comment by bennettforkner** [comment]
Thank you @simonminer , very helpful to hear from someone with experience in accessibility!

In accordance with those above criteria, we should probably put the navigation tabs back into the tab index even when they are not the current tab. Thoughts @EjPlatzer @russtuck ?

**Comment by EjPlatzer** [comment]
The documentation for the `Tabs` component we're using has [an accessibility section](https://mui.com/components/tabs/#accessibility). That links to the [WAI-ARIA guidelines for the `Tabs` interface](https://www.w3.org/TR/wai-aria-practices/#tabpanel) that this component is intended to implement. Those guidelines note the following for keyboard interaction with the the tab list:

> Keyboard Interaction
> For the tab list:
> 
> - <kbd>Tab</kbd>: When focus moves into the tab list, places focus on the active tab element. When the tab list contains the focus, moves focus to the next element in the page tab sequence outside the tablist, which is typically either the first focusable element inside the tab panel or the tab panel itself.
> 
> - When focus is on a tab element in a horizontal tab list:
>   - <kbd>Left Arrow</kbd>: moves focus to the previous tab. If focus is on the first tab, moves focus to the last tab. Optionally, activates the newly focused tab (See note below).
>   - <kbd>Right Arrow</kbd>: Moves focus to the next tab. If focus is on the last tab element, moves focus to the first tab. Optionally, activates the newly focused tab (See note below).

That is, I believe, the rationale for removing the non-active tabs from the tabindex. The intention is that <kbd>Tab</kbd> will focus the active tab and allow arrow key navigation within the tabs list, but hitting tab again will move to the next element outside the tab list.

The above accessibility guidelines do make it clear that we aren't providing the proper aria labels to the tabs and tabpanels, which needs to be fixed, but that is a separate issue from keyboard navigation.
